client/member.go

```diff
@@ -228,12 +228,50 @@ func (c *Client) ResumeMember(userGuid, memberGuid string, challenges []*models.
 	return parseMemberResponse(response)
 }
 
-func (c *Client) ListMemberTransations(userGuid, memberGuid string) ([]*models.Transaction, error) {
+func (c *Client) ListMemberAccounts(userGuid, memberGuid string) ([]*models.Account, error) {
 	if userGuid == "" || memberGuid == "" {
 		return nil, MissingGuid
 	}
 
-	apiEndpointUrl := c.ApiURL + "/users/" + userGuid + "/members/" + memberGuid + "/transactions"
+	apiEndpointUrl := c.ApiURL + "/users/" + userGuid + "/members/" + memberGuid + "/accounts"
+	response, err := Get(apiEndpointUrl, c.defaultHeaders())
+	if err != nil {
+		return nil, err
+	}
+	defer response.Body.Close()
+
+	if err := parseResponseErrors(response.StatusCode); err != nil {
+		return nil, err
+	}
+
+	buffer := new(bytes.Buffer)
+	buffer.ReadFrom(response.Body)
+	bufferStr := buffer.String()
+
+	if response.StatusCode == 200 {
+		accountsResponse := &models.AccountsResponse{}
+		json.Unmarshal([]byte(bufferStr), accountsResponse)
+		return accountsResponse.Accounts, nil
+	}
+
+	return nil, makeGenericError(response.StatusCode, bufferStr)
+}
+
+func (c *Client) ListMemberTransactions(userGuid, memberGuid, fromDate, toDate string) ([]*models.Transaction, error) {
```

Comment:
Same thing here, we shouldn't change this interface, we should add two methods:

```go
func (c *Client) ListMemberTransactions(userGuid, memberGuid) ([]*models.Transaction, error) {
    return ListMemberTransactionsWithDateRange(userGuid, memeberGuid, "", "")
}
// and

func (c *Client) ListMemberTransactionsWithDateRange(userGuid, memberGuid, fromDate, toDate string) ([]*models.Transaction, error) {
```
You really should never change the public interface of a library if you don't need to.